mod/ckeditor/views/default/elgg/ckeditor.js

```diff
@@ -1,25 +1,97 @@
-define(function(require) {
+/**
+ * This module can be used to bind CKEditor to a textarea
+ * <code>
+ *	  require(['elgg/ckeditor'], function(editor) {
+ *	      editor.bind('textarea');
+ *	  });
+ * </code>
+ * 
+ * @module elgg/ckeditor
+ */
+define(function (require) {
 	var elgg = require('elgg');
+	require('elgg/init');
 	var $ = require('jquery');
 	require('jquery.ckeditor');
 	var CKEDITOR = require('ckeditor');
-	var config = require('elgg/ckeditor/config');
```

Comment:
It would be inlined in `default/ckeditor/init.php`. The old module depended on `elgg/init` anyway.